sssom/writers.py

```diff
@@ -303,13 +304,75 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
       - ConcpetMap::SSSOM mapping spreadsheet: https://docs.google.com/spreadsheets/d/1J19foBAYO8PCHwOfksaIGjNu-q5ILUKFh2HpOCgYle0/edit#gid=1389897118
 
     TODOs
-    todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
-    TODO: Add additional fields from both specs
+     todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
+     TODO: Add additional fields from both specs
      - ConceptMap spec fields: https://www.hl7.org/fhir/r4/conceptmap.html
       - Joe: Can also utilize: /Users/joeflack4/projects/hapi-fhir-jpaserver-starter/_archive/issues/sssom/example_json/minimal.json
-     - SSSOM more fields:
-     - prefix_map
-     - SSSOM spec fields: https://mapping-commons.github.io/sssom/Mapping/
+     - SSSOM https://mapping-commons.github.io/sssom/Mapping/
+        author_id,?
+        author_label,?
+        comment,group.element.target.comment
+        confidence,?
+        creator_id,?,?,See: #1
+        creator_label,?,?,See: #1
+        license,copyright~,?,#1: If there is any variation for any records in a MappingSet, this may need to be a group.element.target.extension
```

Comment:
a lot duplication here - should be one SoT

Reply:
Im usually a pretty DRY guy; I'll take a look at what I did (this is a year old). Maybe it's still just in the draft phases.

@cmungall You're the only one I know who uses acronyms that I can't seem to find answers for on Google. SoT?

Reply:
Still not sure what SoT is. But it looks like these are under my TODO's in the docstring for `to_fhir_json()`. I _do_ have [a google sheet where I'm managing SSSOM to FHIR mappings](https://docs.google.com/spreadsheets/d/1J19foBAYO8PCHwOfksaIGjNu-q5ILUKFh2HpOCgYle0/edit#gid=913661924). You'll see it looks similar to the CSV that I deleted. But it's meant more for administrative / tracking purposes. Not to be used as a mapping language. I could potentially move these comments into there (if you guys want), but I don't know that causing harm there right now. Basically I'm just tracking each field to make sure it gets utilized, and marking down any issues I foresee.

Reply:
Ah, looking back at this now, I see you mean "Source of Truth". Makes sense. I went ahead and removed this comment, and will use the google sheet to track my progress instead.